frontend/ui/app/[locale]/home/(about)/AboutSection.tsx

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import { useTranslations } from "next-intl";
+import styles from "./AboutSection.module.css";
+import Image from "next/image";
+
+const AboutSection = () => {
+  const t = useTranslations();
+
+  return (
+    <section>
+      <div className={styles.aboutContainer}>
+        <section>
+          <h1>{t("Index.about.about_heading")}</h1>
+          <div style={{ whiteSpace: "pre-line" }} className={styles.textFormating}>{t("Index.about.about_description")}</div>
+        </section>
+        <section>
+          <Image src={"/assets/cat_about.png"} alt={"White cat"}
+            width={600}
+            height={450}>
+          </Image>
+        </section>
+      </div>
+      <div className={styles.featureContainer}>
+        <section className={styles.container}>
+          <h2>{t("Index.about.first_title")}</h2>
+          <svg
```

Comment:
`Suggestion` to make it as a standalone component, so it does not bloat up the about section. Pasting a named component like <StarIcon/> is also more human-readable then an SVG path